src/Candidate.js

```diff
@@ -1,31 +1,44 @@
 import React from 'react';
 import styled from 'styled-components';
 import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+
 const url =
   'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote?';
 
-function CandidateVotes({ candidate, flipVoteFlag, rank }) {
+function CandidateVotes({ candidate, flipVoteFlag, rank, loginCookie }) {
```

Comment:
리액트 사용할 때 어느정도 컨벤션은 지켜주는게 필요해

1) 컴포넌트 이름(`CandidateVotes`)과 파일명(`Candidate.js`)와 일치시키기
2) 컴포넌트 이름은 더 구체적으로(`Candidate` -> `CandidateRow`)
3) 컴포넌트 최상단 컨테이너는 `Wrapper` 혹은 `Container`로 명명하기
- 이건 왜냐면 `CandidateContainer` 같은 컴포넌트 이름에 의존적으로 지어버리면 컴포넌트 이름을 변경할때마다 최상단 styled-component 컨테이너의 이름도 바꿔줘야 하기 때문!

```suggestion
function CandidateRow({ candidate, flipVoteFlag, rank, loginCookie }) {
```